src/css/utilities/_colors.scss

```diff
@@ -0,0 +1,38 @@
+// Color Utilities
+//
+// Styleguide Utilities.Colors
+
+$text-colors: (
+"black": $color-black,
+"gray-tint": $color-gray-tint,
+"gray-shade": $color-gray-shade,
+"white": $color-white,
+"green-tint": $color-green-tint,
+"green": $color-green,
+"green-shade": $color-green-shade,
+"blue-tint": $color-blue-tint,
+"blue": $color-blue,
+"blue-shade": $color-blue-shade,
+"pink-tint": $color-pink-tint,
+"pink": $color-pink,
+"pink-shade": $color-pink-shade,
+"sand": $color-sand,
+"red": $color-red,
+"yellow-tint": $color-yellow-tint,
+"yellow": $color-yellow,
+"yellow-shade": $color-yellow-shade
+);
+
+@each $name, $var in $text-colors {
+  .u-text-#{$name} {
+    color: $var;
+  }
+
+  .u-background-#{$name} {
+    background-color: $var;
+  }
+
+  .u-fill-#{$name} {
+    fill: $var;
```

Comment:
What are your thoughts on using `!important` on these utilities? I popped a fill class on the logo, but because they're imported early on, it was superseded by `c-logo__icon`.

Reply:
I'd prefer to keep it off. I don't mind occasionally using color declarations outside of scope the utility classes, so long as there's a specific design need.